ocs_ci/ocs/ui/base_ui.py

```diff
@@ -184,6 +184,8 @@ def __init__(self, driver):
         super().__init__(driver)
         ocp_version = get_ocp_version()
         self.page_nav = locators[ocp_version]["page"]
+        if Version.coerce(ocp_version) >= Version.coerce("4.8"):
+            self.generic_loc = locators[ocp_version]["generic"]
```

Comment:
You can split the relevant dictionaries.
for example:
https://github.com/red-hat-storage/ocs-ci/blob/bc6b37b1c51a4002fc1a166f4d1994647b2c3a1e/ocs_ci/ocs/ui/views.py#L246